store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedGraphStorage.java

```diff
@@ -333,7 +333,7 @@ private void addToCache(final GraphSerialisable newGraph, final FederatedAccess
             try {
                 federatedStoreCache.addGraphToCache(newGraph, access, false);
             } catch (final OverwritingException e) {
-                throw new OverwritingException((String.format("User is attempting to overwrite a graph within the cacheService. GraphId: %s", newGraph.getGraphId())));
+                throw new OverwritingException((String.format("User is attempting to overwrite a graph within the cacheService. GraphId: %s", newGraph.getGraphId())), e);
```

Comment:
I can't remember but there is chance that e wasn't thrown incase it reveals access information from the cache?

Reply:
I'm not sure. I've reverted the change to include this and added a suppression instead - it's not really adding much.